app/cache/cache-service.js

```diff
@@ -0,0 +1,32 @@
+const NodeCache = require('node-cache');
+const { Logger } = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('CacheService');
+
+class Cache {
+
+  constructor(name, ttlSeconds, checkPeriodSeconds) {
+    this.cache = new NodeCache({ stdTTL: ttlSeconds, checkperiod: checkPeriodSeconds });
```

Comment:
thinking now of a potential issue with this simple TTL strategy. 
Every token has a TTL itself.
It's possible that we are storing expired tokens in the cache and returning them for usage. 
To avoid this issue we should set the TTL on the single token entry to be the lowest between the cache default TTL and the token's TTL. 
This problem is minimised by setting a small cache TTL tough. Not saying we need to change this but let's start a discussion

Reply:
Reduced cache TTL to 10 mins as discussed 👍